great_expectations/validator/validator.py

```diff
@@ -1800,6 +1801,9 @@ def validate(
                 etc.).
             only_return_failures (boolean): \
                 If True, expectation results are only returned when ``success = False`` \
+            checkpoint_name (str): \
+                Name of the Checkpoint which invoked this Validator.validate() call against an Expectation Suite. \
+                It will be added to `meta` field of the returned ExpectationSuiteValidationResult.
```

Comment:
Nitpick - Could we make this `(str or None)` or `(Optional[str])` to adhere to the rest of the pattern set by the docstring?